Lombiq.Hosting.MediaTheme.Deployer/License.md

```diff
@@ -0,0 +1,15 @@
+Copyright © 2022, [Lombiq Technologies Ltd.](https://lombiq.com)
```

Comment:
Please use the license file given in the 3rd point [here](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2609840133/Preparing+a+.NET+project+to+be+published+on+NuGet)